lexibank_ids.py

```diff
@@ -210,6 +256,32 @@ def cmd_makecldf(self, args):
             lg_org_value = defaultdict(str)
             entries_list = list(entries)
 
+            if lg_id in edited_data:
```

Comment:
Having one `add_form` block so distant, nested within an `if`, from another `add_form` block feels ... weird. Could this be refactored? Also all of the special-case distinctions below make this somewhat hard to read.

Reply:
I really, really see your point here! I decided to do in that way since the import via xlsx is clean without any special cases and without any curated data from etc/ids.all.csv. To incorporate edited_data into the normal for loop over entries_list would end up in having much more if clauses and the readability would be decreased even more. One idea I have would be to generate clean *.csv and *.md files (as for the xlsx files) within the download command. By doing so the makecldf command would be clean and short. On the other hand, how often will such case occur with edited data, since we add new data via separate repos (I mean cost-benefit factor)?

Reply:
I understand, thanks for explaining. Yes, I agree, having this offloaded into `download` also doesn't seem like the best idea.